### DISCUSSION TITLE
I want to use same vault I use in Mac with the IPad

### DISCUSSION THREAD

**Discussion by ecominguez** [remotely-save/remotely-save#359] [Q&A] [answered]
Hi, I already installed in my IPad and I created a vault in dropbox, same name I use in my mac, thinking that using same name will connect to the same vault. It didn’t happened. After reading here, I saw that the vault I created with the IPad, is in applications folder…. instead of taking the folder I have. 
I didn’t find a way to let remote-save knows the exact Dropbox path of my already existing vault.

Is it possible to use/access with the mac and the IPad to the same vault? (I can imagine that if I move my already created vault with my mac, to the one created with the IPad… it could work, but I prefer to have it where it already is)

Thank you a lot. 
Great plugin
Eduardo

**Comment by jthvedt** [comment]
I ran into the same problem. The method I used seems sketchy, so if you do it, use extreme caution, back everything up, etc.. Also I used OneDrive, not Dropbox, and I wanted to sync between an iPhone, an iPad, and Windows. This all may cause different behavior from what you see in your setup. Enough with the disclaimers, but seriously -- this all seems a bit wobbly.

This is all based on the David Sparks mantra that an Obsidian vault is just a folder full of markdown files.

It seems like there should be a better way, but I sure couldn't find it.

Anyway, here's what I had, and what I did: I had created one vault each on the iPad and the iPhone. I had the plug-in installed on all three devices. Windows could see both vaults, but each Apple device could only see the vault created locally.

I made sure to shut down Obsidian everywhere but the device I was changing, and did a manual remotely-save sync at every step. On the iPad, I used the Files app to copy the OneDrive folder (OneDrive > Files > Apps > remotely-save > [name of vault I can't see from here])  to the local Obsidian folder. Now I could open that vault on the iPad. I made a minor test change to a page to make sure the plugin was still syncing.

Then on the iPhone, I did the reverse: copied the iPad's vault folder on OneDrive to the local Obsidian folder.

It's all working so far. Fingers crossed...

**Reply by jthvedt** [reply]
Interesting: somebody [here](https://github.com/remotely-save/remotely-save/discussions/267) simply created an empty vault with the same name, but that didn't work for me. Possibly I didn't do it quite right; I didn't spend a whole lot of time on it.

**Reply by jthvedt** [reply, marked as answer]
The solution became clear once I thought about this for a while. The key was realizing that plugins are vault-specific. Here's how to do it:

- create your new vault on one device. Could be a Mac, could be Windows, another iPad, an iPhone, wherever.
- Install, enable, and configure the Remotely Save plugin for that vault on that first device. Run a sync.
- On the other iPhone or iPad, create a new vault with exactly the same name.
- Install, enable, and configure the Remotely Save plugin for that vault on that second device. Run a sync.

That's it.

Later, I found this comment confirming this method: https://github.com/remotely-save/remotely-save/discussions/135#discussioncomment-2977413
